processes/presenting-at-conferences.rst

```diff
@@ -0,0 +1,154 @@
+#########################
+Presenting at Conferences
+#########################
+
+This is our playbook for presenting on behalf of DM at both project and public meetings.
+
+.. seealso::
+
+   Conference proceedings papers are coordinated by the LSST Project Publication Board.
+   See :doc:`publication-policy` for more information.
+
+.. _presenting-interaction-report:
+
+Interaction report
+==================
+
+As DM team members, we work within an NSF and DOE funded MREFC construction project.
+It's important that we're consistent with the Project when we speak in public and with government officials.
+As you're preparing your presentation:
+
+- `File an interaction report <https://project.lsst.org/interactions>`__. Interaction reports are required when presenting at a public meeting or with government officials.
+- Use and refer to the Project's `pre-approved material <https://project.lsst.org/preapproved>`__ for presentations, including schedule information.
+- Use the Project's `Key Numbers <https://confluence.lsstcorp.org/display/LKB/LSST+Key+Numbers>`__.
+
+.. _presenting-templates:
+
+Slide templates and stock
+=========================
+
+If the meeting doesn't have a specific slide template, use the `Project's templates and stock slides <https://project.lsst.org/documents/stock-slides-templates>`__.
+
+The Project `image and video gallery <https://www.lsst.org/gallery/image-gallery>`__ is also an excellent source of content.
+
+.. _presenting-slack:
+
+Slack during the meeting
+========================
+
+You might want to create a dedicated Slack channel for the meeting you're attending.
+Use this channel to coordinate with other on-site LSST folks and to live-blog for the rest of the team.
+Here's how to make the channel:
+
+1. `Create a channel <https://get.slack.help/hc/en-us/articles/201402297-Create-a-channel>`__ on https://lsstc.slack.com. Use a ``meeting-`` prefix for the channel name.
+   For example: ``meetings-adass-2016``.
+2. Announce the channel in `#lsst-newchannels <https://lsstc.slack.com/archives/lsst-newchannels>`__.
+   You might want to also mention the channel in `#dm <https://lsstc.slack.com/archives/dm>`__ since `#lsst-newchannels <https://lsstc.slack.com/archives/lsst-newchannels>`__ is lightly followed.
+
+.. _presenting-report:
+
+Summarize the meeting on the Community forum
+============================================
+
+Once the meeting is over, it's a great idea to write down what happened.
```

Comment:
For what it's worth, in doing so you'll close [DM-6499](https://jira.lsstcorp.org/browse/DM-6499). Thank you!

And it's not true that nobody has ever filed such a ticket: I refer you to [DM-6716](https://jira.lsstcorp.org/browse/DM-6716). I make no further comment!